### PR TITLE
Update Gigabit T&C, link to 2016 Proposal Template

### PR DESCRIPTION
**Pull request by alanmoo**
## Description
Update Gigabit Community Fund Terms and Conditions

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1286645

## Testing
Content-only changes. gulp test passes.

## Checklist
- [ ] Requires l10n changes.
- [ ] Related functional & integration tests passing.

Closes https://github.com/mozilla/learning.mozilla.org/issues/1922

### CODE REVIEW

File: bedrock/gigabit/templates/gigabit/apply.html [hunk 1]
```diff
@@ -151,7 +151,7 @@ <h1 class="large center">{{ _('Apply') }}</h1>
             {% endtrans %}
             </li>
             <li>
-            {% trans url_handy_template='//docs.google.com/document/d/1XC8SH5Th-8_12kq7L3trtV31fn36HEj7HM7g6zzuM9g/edit' %}
+            {% trans url_handy_template='//drive.google.com/file/d/0B9NqWPTRsfNXNjlKT0tMVlcySlhfZmVlQTBINXVrN0FSVlI0/view' %}
```
Comment (alexgibson) [inline bedrock/gigabit/templates/gigabit/apply.html]: Let's make this link `https://` by default

Comment (alanmoo) [inline bedrock/gigabit/templates/gigabit/apply.html]: WFM, fixed.

**Comment by alexgibson** [comment]
Rest of the changes LGTM, thanks @alanmoo r+

**Comment by alexgibson** [comment]
@alanmoo before we merge, could you please reference the bug number ([bug 1286645](https://bugzilla.mozilla.org/show_bug.cgi?id=1286645)) in your commit message? thanks.

**Comment by alanmoo** [comment]
Thanks, @alexgibson. Out of curiosity, what does the deploy schedule look like for this?

**Comment by alexgibson** [comment]
We don't usually deploy of Friday's unless it's really urgent, so I would expect this to go out on Monday.

**Pull request merged**